### ISSUE TITLE
Add a Copy button beside Sample Input

### ISSUE THREAD

**Issue by prabowo02** [ia-toki/judgels#284]
Just like CodeForces and AtCoder, a copy button for every sample input and sample output

**Comment by fushar** [comment]
Makes sense. Need to add some kind of id / class to the HTML... or somehow we "parse" `Sample Input` from description
